ansible/playbook.yml

```diff
@@ -0,0 +1,14 @@
+- name: install uptime kuma with nginx connected
+  hosts: all
+
+  vars:
+    pip_install_packages:
+      - name: docker
+    docker_compose_version: "v2.0.1"
```

Comment:
docker-compose 2.0.1 is not installable via PIP. They migrated from Python to Golang for v2.x.x